scripts/xtensa-build-all.sh

```diff
@@ -6,7 +6,7 @@
 set -e
 
 SUPPORTED_PLATFORMS=(byt cht bdw hsw apl skl kbl cnl sue icl jsl \
-                    imx8 imx8x imx8m tgl tgl-h)
```

Comment:
This patch should only contain the change for `scripts/xtensa-build-all.sh .

CMakeLists and Kconfig should be updated in previous patch together with adding ``*c`` and ``*h`` files.

Reply:
OK. will modify this.